tests/unit/CurvilinearGridLineMirrorTests.cpp

```diff
@@ -0,0 +1,121 @@
+#include <gtest/gtest.h>
+
+#include <MeshKernel/CurvilinearGrid/CurvilinearGrid.hpp>
+#include <MeshKernel/CurvilinearGrid/CurvilinearGridLineMirror.hpp>
+#include <TestUtils/MakeCurvilinearGrids.hpp>
```

Comment:
is there a test where the new gridline is not added (AddGridLineAtBoundary returning false)?